margaretupshur/richmond.md

```diff
@@ -0,0 +1,11 @@
+# Richmond
+
+### Restaurants
+- Restuarant 1
+- Restaurant 2
+- Really good restaurant
```

Comment:
@margaretupshur oh man, I love Really Good Restaurant! Their entree and side dish is amazing!